warehouse/query-core/src/main/java/datawave/query/jexl/visitors/LiteralNodeSubsetVisitor.java

```diff
@@ -42,4 +45,47 @@ public Object visit(ASTEQNode node, Object data) {
         return literals;
     }
     
+    // Descend through these nodes
+    @Override
+    public Object visit(ASTJexlScript node, Object data) {
+        node.childrenAccept(this, data);
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTAndNode node, Object data) {
```

Comment:
The short circuit visitor does not short circuit and, or, reference, or expression nodes.  I don't think you need to override those methods here.

Reply:
For some reason, I thought BaseVisitor blocked but must have been an oversight.